### ISSUE TITLE
mildom.com

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#4656]
### Basic Information

**Test link:** `https://www.mildom.com/10212123`
**Category:** breakage
**Reported from:** Japan
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:80.0) Gecko/20100101 Firefox/80.0
**Extension:** Nano Adblocker 1.0.0.151

**Comment by krystian3w** [comment]
@Yuki2718? Maybe nswf.

**Comment by jspenguin2017** [comment]
Why do you think it's NSFW?

**Comment by krystian3w** [comment]
Becaue I saw only 5 sec but not all transmission.

We do not know how loose rules accept this alternative to twitch is no accepted.

**Comment by Yuki2718** [comment]
Yup, safe for work. I don't see any breakage with uBO default lists + AGJPN.

**Comment by jspenguin2017** [comment]
I don't see any obvious breakage neither.
